src/main/java/frc/robot/Constants.java

```diff
@@ -18,6 +19,23 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+
+  public static final class AutoConstants {
+    public static final double kMaxSpeedMetersPerSecond =
+        DriveConstants.kMaxSpeedMetersPerSecond / 4;
+    public static final double kMaxAngularSpeedRadiansPerSecond =
+        DriveConstants.kMaxAngularSpeed / 10;
```

Comment:
Why setting the constants in DriveConstants and then modifying before using?  Use the /4 and /10 values in the constants instead of here.